### DISCUSSION TITLE
想讓繁中服能支援自動基建

### DISCUSSION THREAD

**Discussion by Arcelibs** [MaaAssistantArknights/MaaAssistantArknights#1415] [General]
截圖的部分可以幫忙~目前需要有哪些資源呢?

**Comment by MistEO** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/discussions/1329 可以先参照这个里面日服的资源截一些。因为我们还没有开始适配日服，不确认他这个完不完整，不过简单看了一下起码没有多余的，先把这部分弄出来吧。感谢~

**Reply by MistEO** [reply]
具体的操作方法可以参考 https://github.com/MaaAssistantArknights/MaaAssistantArknights/discussions/1212#discussioncomment-3176596

**Reply by Arcelibs** [reply]
> 具体的操作方法可以参考 [#1212 (comment)](https://github.com/MaaAssistantArknights/MaaAssistantArknights/discussions/1212#discussioncomment-3176596)

問下，有些圖旁邊有個Mini，是指什麼意思

然後CropRoi工具 他是把剪好的圖自己放入一個新建的.src中去剪圖?

**Reply by MistEO** [reply]
- mini 是点击基建右上角蓝色的通知后，再取消，这时基建会缩小到最小的状态
- src 是把原图放进去然后一个裁剪的小工具，你要是有 ps 之类的其他裁剪工具可以不用这个

**Reply by Arcelibs** [reply]
> #1329 可以先参照这个里面日服的资源截一些。因为我们还没有开始适配日服，不确认他这个完不完整，不过简单看了一下起码没有多余的，先把这部分弄出来吧。感谢~

其實我也不曉得有哪些，照著有看過的印象稍微截了一些圖提供給大佬參考下，謝謝

[TW.Server.zip](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/9253011/TW.Server.zip)

**Comment by MistEO** [comment]
日服基建 PR，可以参考下 https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/2758
